test/fob/computed_select_test.exs

```diff
@@ -0,0 +1,98 @@
+defmodule Fob.ComputedSelectTest do
+  use Fob.RepoCase
+  alias Fob.Cursor
+  alias Ecto.Multi
+
+  describe "query with single table" do
+    setup do
+      [schema: SimplePrimaryKeySchema, repo: Fob.Repo]
+    end
+
+    setup c do
+      records = for n <- 1..20, do: %{id: n}
+
+      Multi.new()
+      |> Multi.insert_all(:seeds, c.schema, records)
+      |> c.repo.transaction()
+
+      :ok
+    end
+
+    test "when selecting with computed query, we can get each page", c do
+      cursor =
+        Cursor.new(
+          from(
+            s in c.schema,
+            select: %{
+              id: s.id,
+              computed_column: 1 >= 5
+            },
+            order_by: [asc: s.id]
+          ),
+          c.repo,
+          _initial_page_breaks = nil,
+          5
+        )
+
+      assert {records, cursor} = Cursor.next(cursor)
```

Comment:
Iunno how I feel about the asserts depending on one another.  Nowait, I dun like it.

```
# pseudosetup
actual_pages = some_recursive_page_counting_function(bounded_depth_call: 10)

# an expectation I expected of a property of the test
assert actual_pages === 4
```

Obviously, not terribly helpful with the hand  waviness of counting pages that might go on forever and still making the test useful.

Reply:
agreed, I find I write a lot of tests that I really wish I could just `refute_raise`, but I'm sure I can find a better allegory for the asserts

Reply:
Perf, that does help, ty.